owncloudData/src/test/java/com/owncloud/android/data/files/repository/OCFileRepositoryTest.kt

```diff
@@ -198,6 +198,21 @@ class OCFileRepositoryTest {
         }
     }
 
+    @Test
+    fun `getFilesForAccount returns a list of OCFile`() {
+        every {
+            localFileDataSource.getFilesForAccount(OC_ACCOUNT_NAME)
+        } returns listOf(OC_FILE)
+
+        val result = ocFileRepository.getFilesForAccount(OC_ACCOUNT_NAME)
+
+        assertEquals(listOf(OC_FILE), result)
+
+        verify(exactly = 1) {
+            localFileDataSource.getFilesForAccount(OC_ACCOUNT_NAME)
+        }
+    }
+
```

Comment:
Tests for `OCFileRepository` are not refactored yet, but ok

Reply:
Added the empty list test as well